src/models.py

```diff
@@ -904,11 +906,17 @@ def _pair_sentence_forward(self, batch, task, predict):
         classifier = self._get_classifier(task)
         if self.use_bert:
             sent, mask = self.sent_encoder(batch['inputs'], task)
-            logits = classifier(sent, mask)
+            if isinstance(task, WiCTask):
```

Comment:
Ideally, we should just create different fwd functions for different tasks to minimize the number of places we need these special cases. This is totally fine/standard for now, though, but try to add more comments explaining why each of these special cases is necessary.

Reply:
Yea I agree that having these special cases aren't great long term :/ I'll add a comment explaining for now.